SysBot.Pokemon.YouTube/Helpers/Scopes.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System.Collections.Generic;
+using YouTube.Base;
+
+namespace SysBot.Pokemon.YouTube
+{
+    class Scopes
+    {
+        public static List<OAuthClientScopeEnum> scopes = new List<OAuthClientScopeEnum>()
```

Comment:
Should be readonly; and why does it have to be a list rather than array? If it's not mutated, it should be an IReadOnlyList<T>